opendevin/runtime/sandbox.py

```diff
@@ -14,19 +12,8 @@ class Sandbox(ABC, PluginMixin):
 
     def __init__(self, config: SandboxConfig):
         self.config = copy.deepcopy(config)
-        for key in os.environ:
-            if key.startswith('SANDBOX_ENV_'):
-                sandbox_key = key.removeprefix('SANDBOX_ENV_')
-                self.add_to_env(sandbox_key, os.environ[key])
-        if config.enable_auto_lint:
-            self.add_to_env('ENABLE_AUTO_LINT', 'true')
         self.initialize_plugins: bool = config.initialize_plugins
 
-    def add_to_env(self, key: str, value: str):
-        self._env[key] = value
-        # Note: json.dumps gives us nice escaping for free
-        self.execute(f'export {key}={json.dumps(value)}')
-
```

Comment:
Will these removal influence the other sandbox?

Reply:
We move this logic to `Runtime`-level, and implement it there